### ISSUE TITLE
add Git solutions comparison tab on NostrGit

### ISSUE THREAD

**Issue by rusteroobs** [NostrGit/NostrGit#222]
Hello,

A random idea would be to compare different git solutions with NostrGIt.
- NostrGIt x GitHub
- NostrGit x Radicle
- NostrGit x GitLab
- NostrGit x Sourcehut
- NostrGit x Sourceforge
- NostrGit x AzureDev

**Comment by AntwanSherif** [comment]
I think this will be useful for new users/orgs who will be evaluating whether to use NostrGit or not (some time in the future once we release). However, I don't think we need to have an exhaustive list with all the Git solutions out there.

**Comment by eflixandrols** [comment]
> I think this will be useful for new users/orgs who will be evaluating whether to use NostrGit or not (some time in the future once we release). However, I don't think we need to have an exhaustive list with all the Git solutions out there.

thanks for feedback! So,  I was able to gather more technical information about different git solutions

![image](https://user-images.githubusercontent.com/130712783/231904183-9a4f5bb2-46ed-41a8-8d48-99d836334fb1.png)

**Comment by cypherhoodlum** [comment]
Good idea. I added the "later" label because the NostrGit project is in its infancy right now and the basic core stuff needs to be designed and implemented first for us to know how NostrGit compares to other git services. Much to do!

**Comment by marcelolucario** [comment]
Hi cypherhoodlum and AntwanSherif , thanks for feedback - you are amazing.

**Comment by gorlask** [comment]
> Good idea. I added the "later" label because the NostrGit project is in its infancy right now and the basic core stuff needs to be designed and implemented first for us to know how NostrGit compares to other git services. Much to do!

I forgot to add to the list other git solutions that may someday be diagnosed. For example, 

> NostrGIt x GitHub
> NostrGit x Radicle
> NostrGit x GitLab
> NostrGit x Sourcehut
> NostrGit x Sourceforge
> NostrGit x AzureDev
> NostrGit x [cjb/GitTorrent](https://github.com/cjb/GitTorrent)
> NostrGit x [spearson78/gitnostr](https://github.com/spearson78/gitnostr)
> NostrGit x [git-ftp/git-ftp](https://github.com/git-ftp/git-ftp)
> NostrGit x nostrya
> NostrGit x gitea
> NostrGit x gogs
> NostrGit x darcs
> NostrGit x Codeberg
> NostrGit x Bitbucket
> NostrGit x GnuSavannah
> NostrGit x [pijul](https://nest.pijul.com/pijul)

**Comment by melvincarvalho** [comment]
Should add also 

- [ngit](https://gitworkshop.dev/)
- [hornet](https://github.com/orgs/HORNET-Storage/repositories)
